### PR TITLE
Add Content Security Policy Handler Chart

### PR DESCRIPTION
**Pull request by gearnode**

### CODE REVIEW

**Review by lukkor — APPROVED** [review]

File: csp-handler/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Redis
```
Comment (clook) [inline csp-handler/README.md]: CSP Handler

File: csp-handler/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Prerequisites
+- Kubernetes 1.6+ with Beta APIs enabled
+
+## Install
+To install the chart with the release name my-release:
+```sh
+helm install --name my-release jobteaser/csp-handler
+```
+The command deploys Redis on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
```
Comment (clook) [inline csp-handler/README.md]: s/Redis/CSP Handler/

File: csp-handler/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+> **Tip**: List all releases using `helm list`
+
+
+## Configuration
+The following tables lists the configurable parameters of the Redis chart and their default values.
```
Comment (clook) [inline csp-handler/README.md]: s/Redis/CSP Handler/

File: csp-handler/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+### Basic
+| Parameter                     | Description                                     | Default                                                    |
+| -----------------------       | ---------------------------------------------   | ---------------------------------------------------------- |
+| `image.repository`            | `redis` image repository and tag                | `redis:3.2`                                                |
```
Comment (clook) [inline csp-handler/README.md]: s@redis:3.2@jobteaser/csp-handler:0.1.0@

File: csp-handler/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+| `image.tag`                   | The tag to use of the image                     | `0.1.0`                                                    |
+| `image.pullPolicy`            | The pull policy of the image                    | `IfNotPresent`                                             |
+
+### Expert
+
+| Parameter                     | Description                                     | Default                                                    |
+| -----------------------       | ---------------------------------------------   | ---------------------------------------------------------- |
+| `service.port`                | TCP port use by the redis-server                | `80`                                                       |
```
Comment (clook) [inline csp-handler/README.md]: s/redis-server/csp-handler/

**Review by clook — APPROVED** [review]
LGTM appart from the few copy/paste stuffs.

**Pull request merged**